game_start.py

```diff
@@ -0,0 +1,28 @@
+class GameStart:
+
```

Comment:
IMO it seems overly complex that we have to instantiate this class and then call a method in order to retrieve a string with a fixed value and pass it onto the console. Are there any alternative ways of storing these types of constants that could streamline that process?

Reply:
yeaaah. i totally agree. i think it was a bad call to to take it out of game class for now. mostly because i think i was accounting for a menu class which i know is a really bad reason to make a decision like this. I could've just relegated to a method in game class

Reply:
imo it doesn't seem like a gameplay responsibility to display a greeting?

Reply:
Yeah I agree it doesn't belong in gameplay. That's why I took it out of there. But I couldn't really think of where to add it. Any suggestions would definitely be appreciated :)  I'm honestly completely lost on I/O as I've mentioned before. And I haven't really found any clarification online on how to implement console correctly. I seriously feel completely lost on how to handle messaging.

Reply:
i mean messaging in general should probably be it's own ticket maybe? given that i have no clue how to implement it?